nsqd/topic.go

```diff
@@ -3,15 +3,14 @@ package nsqd
 import (
 	"bytes"
 	"errors"
-	"strings"
-	"sync"
-	"sync/atomic"
-	"time"
-
 	"github.com/nsqio/go-diskqueue"
 	"github.com/nsqio/nsq/internal/lg"
 	"github.com/nsqio/nsq/internal/quantile"
 	"github.com/nsqio/nsq/internal/util"
+	"strings"
+	"sync"
+	"sync/atomic"
+	"time"
```

Comment:
can you put these imports back at the top please?

(If you leave a blank line between them and the github.com/... imports, then gofmt won't sort them together, it will keep them in separate sections.)

Reply:
OK，thank you for your reminding